src/components/InfoSection/InfoSection.styles.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+
+export const InfoSectionWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 127px;
+`;
+export const Info = styled.div`
+  display: flex;
+  width: 738px;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  color: ${({ theme }) => theme.colors.grayBase};
+  font-family: ${({ theme }) => theme.fonts.primary};
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.69;
+  letter-spacing: 0.03px;
+  a {
```

Comment:
Normally I wouldn't like targeting an HTML tag like this. Here it might be OK since you just change an `a` to a `div` or so... But wait a minute, why isn't this inside the global styles?

Reply:
Will move it to a global style. Footer links are a tags as well so will need to change footer color in footer styles then but it makes sense to have global styles links with this blue color globally.